.github/workflows/download-pulumi-cron.yml

```diff
@@ -37,7 +37,7 @@ jobs:
         id: vars
         run: |
           echo "installed-version=$(pulumi version)" >> "${GITHUB_OUTPUT}"
-          echo "expected-version=v$(curl -sS https://www.pulumi.com/latest-version)" >>" ${GITHUB_OUTPUT}"
+          echo "expected-version=v$(curl -sS https://www.pulumi.com/latest-version)" >> "${GITHUB_OUTPUT}"
```

Comment:
The change to this line makes it match all the other `echo "expected-version=[...]` lines in the file (e.g. line 24).

Reply:
oh wow. yeah, good catch